js/data-analyst-agent-v2/README.md

```diff
@@ -0,0 +1,186 @@
+<!--
+---
+id: data-analyst-agent
```

Comment:
might conflict on the site, probably `-v2` or the like? 

more of a teams discussion but do we still want to keep v1 of data analyst in the templates

Reply:
if so, we should probably adjust this README to incl v2